bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/handler/OpenWeatherMapOneCallHandler.java

```diff
@@ -75,9 +84,10 @@ public class OpenWeatherMapOneCallHandler extends AbstractOpenWeatherMapHandler
 
     private @Nullable OpenWeatherMapOneCallAPIData weatherData;
 
-    private int forecastMinutes = 60;
-    private int forecastHours = 24;
-    private int forecastDays = 8;
+    // forecastMinutes, -Hours and -Days determine the number of channel groups to create for each type
+    private int forecastMinutes = 0;
+    private int forecastHours = 0;
+    private int forecastDays = 0;
```

Comment:
Should this be reverted as well?

Reply:
Type of yes.
I have set this to defaults from the Thing config file, makes more sense IMO. Otherwise the Thing will rebuild its channels when reading default config while initizializing.